src/scripts/main.js

```diff
@@ -1 +1,50 @@
 'use strict';
+
+function sliderStart() {
+  const slider = document.querySelector('.slider');
+  const cards = Array.from(document.querySelectorAll('.carousel__item'));
```

Comment:
оно ведь итак должно вернуть массив, не?

Reply:
The Document method querySelectorAll() returns a static (not live) NodeList representing a list of the document's elements that match the specified group of selectors.

NodeList objects are collections of nodes, usually returned by properties such as Node.childNodes and methods such as document.querySelectorAll().

Although NodeList is not an Array, it is possible to iterate over it with forEach(). It can also be converted to a real Array using Array.from().

However, some older browsers have not implemented NodeList.forEach() nor Array.from(). This can be circumvented by using Array.prototype.forEach() — see this document's Example.